.github/workflows/server-build.yml

```
@@ -344,7 +344,11 @@ jobs:
           git lfs pull ./server.jar
           mv ./server.jar ../../../../../server.jar
           cd ../../../../../
-          tar -xzvf ./server.jar
+ #         tar -xzvf ./server.jar
+ 
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)